src/main/scala/com/databricks/spark/xml/util/TypeCast.scala

```diff
@@ -17,12 +17,11 @@ package com.databricks.spark.xml.util
 
 import java.math.BigDecimal
 import java.sql.{Date, Timestamp}
-import java.text.NumberFormat
+import java.text.{NumberFormat, SimpleDateFormat}
 import java.util.Locale
 
 import scala.util.Try
 import scala.util.control.Exception._
-
```

Comment:
I don't know why it's not caught but we need this line .. it's a part of style - see https://github.com/databricks/scala-style-guide#imports